frontend/components/TableContainer/DataTable/DataTable.tsx

```diff
@@ -301,10 +308,20 @@ const DataTable = ({
     }
   }, [selectedDropdownFilter]);
 
+  // track previous sort state
+  const prevSort = useRef<{ id?: string; desc?: boolean }>({
+    id: undefined,
+    desc: undefined,
```

Comment:
```suggestion
    desc: undefined, // desc as in descending
```

When I first looked at this I thought it meant "description" and was ???